roles/wildfly_app_deploy/meta/argument_specs.yml

```diff
@@ -0,0 +1,43 @@
+argument_specs:
```

Comment:
There should be downstream arguments as well, like [this](https://github.com/ansible-middleware/wildfly/blob/main/roles/wildfly_install/meta/argument_specs.yml#L72)